packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -608,14 +425,13 @@
 }
 
 // Graceful degredation for css-only inputs
-
-.mdc-text-field {
-  &:not(.mdc-text-field--upgraded):not(.mdc-text-field--textarea) .mdc-text-field__input {
+.mdc-text-field:not(.mdc-text-field--upgraded) {
```

Comment:
Changing this selector from how it was affects the usage of the `mdc-theme-dark` mixin inside of this block.

Previously, it selected on `.mdc-text-field--theme-dark...` but now it selects on `... .mdc-input--theme-dark`.

You might need to revert this selector change, but maybe passing a root selector to the `mdc-theme-dark` could work?

(I found this regression by diffing the output CSS on master running `npm build` against the output CSS from this branch.)

Reply:
Thanks for finding! I thought I had covered the case when I manually tested, but I will revert.